src/util/importText.js

```diff
@@ -344,85 +343,3 @@ export const importHtml = (thoughtsRanked, html, { skipRoot, state } = {}) => {
     thoughtIndexUpdates,
   }
 }
-
-/** Imports the given text or html into the given thoughts
- *
-  @param preventSetCursor  Prevents the default behavior of setting the cursor to the last thought at the first level
-  @param preventSync       Prevent syncing state, turning this into a pure function.
-  @param rawDestValue      When pasting after whitespace, e.g. pasting "b" after "a ", the normal destValue has already been trimmed, which would result in "ab". We need to pass the untrimmed destination value in so that it can be trimmed after concatenation.
-  @param skipRoot          See importHtml @param.
- */
-export const importText = (thoughtsRanked, inputText, { preventSetCursor, preventSync, rawDestValue, skipRoot } = {}) => {
```

Comment:
It's confusing to have a util file called `importText` that does not contain an `importText` function, given that in the project the convention is to name a file the same as its primary export. If you are going to split out `importText` into its own function`, I recommend renaming this file to its primary export, `importHtml`.

Reply:
Yes that makes sense. I will change the filename to `importHTML`